suitcase/hdf5.py

```diff
@@ -94,8 +95,27 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 compression='gzip',
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
-                    dataset = data_group.create_dataset(
-                        key, data=data, compression='gzip', fletcher32=True)
+                    data = np.array(data)
+                    try:
+                        # save numerical data
+                        dataset = data_group.create_dataset(
+                            key, data=data, compression='gzip', fletcher32=True)
+                    except TypeError:
```

Comment:
I try numerical data first. Otherwise we assume the data is string type, then we deal with 1d or 2d cases of string. That's it.